application/world/services/auto_bible_generator.py

```
@@ -261,17 +261,88 @@ async def generate_and_save(
+        logger = logging.getLogger(__name__)
+        logger.info("Starting background research for premise...")
+
+        # 获取专门用于深度研究的模型 (Research Model)
+        target_model = ""
+        from infrastructure.ai.config.dynamic_settings import DynamicSettingsManager
+        from interfaces.api.dependencies import _build_provider_for_role
+        
+        dyn_config = DynamicSettingsManager().load_config()
+        research_provider = _build_provider_for_role(dyn_config, "research")
+        
+        # 如果单独配置了 research_model，则使用 research_provider，否则回退到主 llm_service
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)